src/projects/TodoList/components/CategoryList/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './style.css';
+
+class CategoryList extends React.Component {
+  render () {
+    let categoriesInfo = this.props.categoriesInfo; // <-- category array of objects
+    let eachTaskCategory = this.props.taskCategories.map((categoryId, i) => {
+      let categoryColour = categoriesInfo.find( category => category.id === categoryId )
```

Comment:
The variable name `categoryColour` is confusing - it looks like it's storing the category object instead of just the colour

Reply:
done